### PR TITLE
First commit of Kawii9

### PR DESCRIPTION
**Pull request by kiwikey**
## Description

Add new keyboard: Kawii9, which is a custom 3x3 keypad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kiwikey/kawii9/config.h [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4b57    // KW - KiwiKey
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/config.h:23]:
```suggestion
#define VENDOR_ID    0x4B57    // KW - KiwiKey
```

File: keyboards/kiwikey/kawii9/config.h [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+#define PRODUCT_ID   0x0303    // 3x3
+#define DEVICE_VER   0x0002
+#define MANUFACTURER KiwiKey
+#define PRODUCT      Kawii9
+#define DESCRIPTION  A fully customizable 3x3 keypad
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/config.h:28]: This is unused.
```suggestion
```

File: keyboards/kiwikey/kawii9/kawii9.h [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_3x3( \
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/kawii9.h:29]:
```suggestion
#define LAYOUT_ortho_3x3( \
```

File: keyboards/kiwikey/kawii9/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_3x3(
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/keymaps/default/keymap.c:25]:
```suggestion
    [_BASE] = LAYOUT_ortho_3x3(
```

File: keyboards/kiwikey/kawii9/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        KC_MUTE,    KC_VOLD,    KC_VOLU,    // Media volume: Mute - Up - Down
+        KC_MPLY,    KC_MPRV,    KC_MNXT,    // Media track control: Play/Pause - Previous Track - Next Track
+        MO(_FN),    LCTL(KC_C), LCTL(KC_V)  // FN - Copy - Paste
+    ),
+	[_FN] = LAYOUT_3x3(
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/keymaps/default/keymap.c:30]:
```suggestion
	[_FN] = LAYOUT_ortho_3x3(
```

File: keyboards/kiwikey/kawii9/info.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
+    "keyboard_name": "Kawii9",
+    "url": "http://kiwikey.vn/kawii9/",
+    "manufacturer": "KiwiKey",
+    "maintainer": "KiwiKey",
+    "width": 3,
+    "height": 3,
+    "key_count": 9,
+    "layouts": {
+        "LAYOUT": {
```
Comment (drashna) [inline keyboards/kiwikey/kawii9/info.json:10]:
```suggestion
        "LAYOUT_ortho_3x3": {
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/kiwikey/kawii9/rules.mk [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/kiwikey/kawii9/rules.mk:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by kiwikey** [comment]
Dear Ryan @fauxpark,
May I know how long does it take to make my pull request merged with the official qmk_firmware? Or there is anything else needed to process it?
This is my first time trying to make my keyboard available in qmk_firmware, there is a high chance that I missed something. Appreciate your help, with many thanks!

File: keyboards/kiwikey/kawii9/info.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
+{
+    "keyboard_name": "Kawii9",
+    "url": "http://kiwikey.vn/kawii9/",
+    "manufacturer": "KiwiKey",
+    "maintainer": "KiwiKey",
+    "width": 3,
+    "height": 3,
+    "key_count": 9,
```
Comment (fauxpark) [inline keyboards/kiwikey/kawii9/info.json:8]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Just this last small change, then since we've got one more approval from Drashna, should be good to merge.

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**